homeassistant/core.py

```diff
@@ -1518,8 +1518,8 @@ class State:
     entity_id: the entity that is represented.
     state: the state of the entity
     attributes: extra information on entity and state
-    last_changed: last time the state was changed, not the attributes.
-    last_updated: last time this object was updated.
+    last_changed: last time the state was changed.
+    last_updated: last time the state or attributes were changed.
```

Comment:
We could also mention something about force_update, but hopefully we'll be able to deprecate that in the not too distant future.